mmdet3d/apis/inferencers/base_det3d_inferencer.py

```diff
@@ -240,7 +287,7 @@ def pred2dict(self, data_sample: InstanceData) -> Dict:
         """
         pred_instances = data_sample.pred_instances_3d.numpy()
         result = {
-            'bboxes_3d': pred_instances.bboxes_3d.tensor.numpy().tolist(),
+            'bboxes_3d': pred_instances.bboxes_3d.tensor.cpu().tolist(),
```

Comment:
numpy() and cpu(), which one is suitable?

Reply:
I run the `.numpy()` and it caused an error on my local device.